how-to/scale/with-ansible.md

```diff
@@ -85,12 +87,37 @@ TiDB 集群可以在不影响线上服务的情况下进行扩容和缩容。
     | node7 | 172.16.10.7 | TiKV2 |
     | node8 | 172.16.10.8 | TiKV3 |
     | node9 | 172.16.10.9 | TiKV4 |
+    
+    - 编辑 hosts.ini
+    ```ini
+    [servers]
+    172.16.10.1
+    172.16.10.2
+    172.16.10.3
+    172.16.10.4
+    172.16.10.5
+    172.16.10.6
+    172.16.10.7
+    172.16.10.8
+    172.16.10.9
+    172.16.10.101
+    172.16.10.102
+
+    [all:vars]
+    username = tidb
+    ntp_server = pool.ntp.org
+    ```
 
 2. 初始化新增节点：
 
     {{< copyable "shell-regular" >}}
 
     ```bash
+    # 在中控机上配置部署机器 SSH 互信及 sudo 规则
```

Comment:
It is recommended to separate comments from commands and add the code snippet `{{< copyable "shell-regular" >}}` for each command, which improves usability.